### PR TITLE
728: add instructions for re-running from scratch

### PR DESCRIPTION
**Pull request by oliver-sanders**
Closes https://github.com/cylc/cylc-doc/issues/393

### CODE REVIEW

File: src/7-to-8/major-changes/play-pause-stop.rst [hunk 1]
```diff
@@ -29,9 +29,34 @@ commands:
...
+
+It is still possible to re-run workflows in-place in the Cylc 7 manner, however,
+this is discouraged.
+To do this remove the workflow database as well as any other evidence of the 
+previous run that is no longer desired:
+
+.. code-block:: bash
+
+   # remove the workflow database, the work, share and log directories
+   cylc clean <id> --rm .serice/db --rm work --rm share --rm log
```
Comment (MetRonnie) [inline src/7-to-8/major-changes/play-pause-stop.rst:54]:
```suggestion
   cylc clean <id> --rm .service/db --rm work --rm share --rm log
```

And can even shorten
```suggestion
   cylc clean <id> --rm .service/db:work:share:log
```

File: src/7-to-8/major-changes/play-pause-stop.rst [hunk 2]
```diff
@@ -29,9 +29,34 @@ commands:
...
+
+   # only remove the worflow database
+   $ cylc clean <id> --rm .serice/db
```
Comment (MetRonnie) [inline src/7-to-8/major-changes/play-pause-stop.rst:57]:
```suggestion
   $ cylc clean <id> --rm .service/db
```

File: src/7-to-8/major-changes/play-pause-stop.rst [hunk 3]
```diff
@@ -29,9 +29,34 @@ commands:
...
+
+Re-Running Workflows
+--------------------
+
+The ``cylc play`` command will always pick up where it left off (a
+:term:`restart`).
+
+If you want to re-run the entire workflow again from the start either:
+
+* Install a new run.
```
Comment (wxtim) [inline src/7-to-8/major-changes/play-pause-stop.rst:43]: Might be worth having a link to how to do this here. The suggestion is slightly generic, you might want to add a new reference at about line 145 of `user-guide/installing-workflows.rst` and link to that.

```suggestion
* :ref:`Install a new run<Using Cylc Install>`.
```

File: src/7-to-8/major-changes/play-pause-stop.rst [hunk 4]
```diff
@@ -29,9 +29,34 @@ commands:
...
+* Or if you want to keep the data from the old run start a :term:`reflow`.
+
+It is still possible to re-run workflows in-place in the Cylc 7 manner, however,
+this is discouraged.
```
Comment (wxtim) [inline src/7-to-8/major-changes/play-pause-stop.rst:47]: I wonder if a warning box with a wee explanation of why it's discouraged might be apposite here?

**Review by wxtim — APPROVED** [review]
Nothing worth blocking you merging.

File: src/7-to-8/major-changes/play-pause-stop.rst (hunk 4 above)
Comment (oliver-sanders) [reply src/7-to-8/major-changes/play-pause-stop.rst:47]: Considered, however, I wanted to keep this topic brief, it's a minor sidenote.

**Pull request merged**